pennylane_lightning/src/util/Util.hpp

```diff
@@ -53,6 +53,75 @@ using CBLAS_LAYOUT = enum CBLAS_LAYOUT {
 #endif
 /// @endcond
 
+namespace Pennylane::Util::Internal {
+constexpr auto countBit1(uint32_t n) -> int {
```

Comment:
Should this not be using `std::size_t` as return type since we are counting values? Same as below,